### PR TITLE
Update the debug page

### PR DESCRIPTION
**Pull request by praneesha**
Update the debug page

### CODE REVIEW

File: en/docs/debug-the-code.md [hunk 1]
```diff
@@ -119,34 +118,61 @@ Follow the steps below to start a remote debug session.
...
+            "debuggeeHost": "127.0.0.1",
+            "debuggeePort": "5005"
+        }
+    ]
+}
+```
+
+### Configuration attributes
+
+The auto-generated `launch.json` file consists of three main configurations named `Ballerina Debug`, `Ballerina Test`, and `Ballerina Remote`. Each configuration supports a different set of attributes, which will be provided via IntelliSense completion suggestions.
```
Comment (NipunaRanasinghe) [inline en/docs/debug-the-code.md:175]:
```suggestion
The auto-generated `launch.json` file consists of three debug configurations named `Ballerina Debug`, `Ballerina Test`, and `Ballerina Remote`. Each configuration supports a different set of attributes, which will be provided via IntelliSense completion suggestions.
```

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

File: en/docs/assets/css/theme.css [hunk 2]
```diff
@@ -111,14 +111,14 @@ html .md-typeset .superfences-tabs > label:hover {
...
-.md-search__input {
+/* .md-search__input {
     background-color: #dedede;
     padding-left: 3rem;
 }
 
 .md-search__input::placeholder {
     color: #404040;
-}
+} */
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/assets/css/theme.css:121]: Fixed via https://github.com/wso2/docs-ballerina-vscode-extension/pull/86.

[the remark this answers is not in the document]
Comment (anupama-pathirage) [reply en/docs/assets/css/theme.css:121]: Seems not fully resolved via above PR as there is more commented code blocks in this PR.

[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/assets/css/theme.css:121]: Will fix via https://github.com/wso2/docs-ballerina-vscode-extension/pull/106.

**Pull request merged**